wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt

```diff
@@ -167,7 +167,8 @@ internal constructor(
     val request = OkHttpUtils.buildStringGetRequest(endpointUrl, authToken)
 
     log.debug {
-      "Anchor account's all transactions request: assetCode = $assetCode, authToken = ${authToken.take(8)}"
+      "Anchor account's all transactions request: assetCode = $assetCode, authToken = ${authToken
```

Comment:
[nits] can you add a (value) type for AuthToken and an extension function for .take(8)? Something like `AuthToken.prettify()`